### PR TITLE
Introduce a flag to disable automatic download

### PR DESCRIPTION
**Pull request by yyoncho**

### CODE REVIEW

**Review by ericdallo — APPROVED** [review]

File: lsp-mode.el [hunk 1]
```diff
@@ -5007,6 +5007,12 @@ In addition, each can have property:
   :group 'lsp-mode
   :type 'boolean)
 
+(defcustom lsp-enable-server-download t
```
Comment (michaelpj) [inline lsp-mode.el:5010]: Maybe put "auto" in the name somewhere? Since this doesn't affect the ability to use e.g. `lsp-install-server`

Comment (ericdallo) [reply lsp-mode.el:5010]: Make sense to me WDYT @yyoncho ?

**Review by jcs090218 — APPROVED** [review]

**Comment by abbec** [comment]
Any ETA on this? :)

File: lsp-mode.el (hunk 1 above)
Comment (yyoncho) [reply lsp-mode.el]: I added the word suggest, because the "auto" is not actually auto.

**Pull request merged**